Unable to install with next@10.2.3

**Describe the bug**
Hey. I just started my new NextJS project and wanted to try NestJS-like decorators within my backend, so I followed documentation and tried to install this package, but got the following error:
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: flaut-nextjs@1.0.0
npm ERR! Found: next@10.2.3
npm ERR! node_modules/next
npm ERR!   next@"^10.2.3" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer next@"10.1.3" from @storyofams/next-api-decorators@1.4.6
npm ERR! node_modules/@storyofams/next-api-decorators
npm ERR!   @storyofams/next-api-decorators@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /home/goodwin/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/goodwin/.npm/_logs/2021-06-13T22_07_17_838Z-debug.log
```
I see that you included `next` in your `peerDependencies` which I'm not really familiar with, but since you are using yarn and I actually remember running into an issue with it's installing dependencies in different maner than npm I can suggest that you meant to use `"next": "^10.1.3"` (10.1.3 is a minimum requirement) instead of `"next": "10.1.3"` (exactly 10.1.3). Please correct me if I'm wrong.

--- a/package.json
+++ b/package.json
@@ -43,9 +43,9 @@
     "reflect-metadata": "^0.1.13"
   },
   "peerDependencies": {
-    "@babel/plugin-proposal-decorators": "7.13.15",
-    "babel-plugin-parameter-decorator": "1.0.16",
-    "next": "10.1.3"
+    "@babel/plugin-proposal-decorators": "^7.13.15",
+    "babel-plugin-parameter-decorator": "^1.0.16",
+    "next": "^10.0.0"
   },
   "devDependencies": {
     "@babel/cli": "7.13.14",

